package.nls.json

```diff
@@ -168,7 +168,7 @@
     "AWS.credentials.statusbar.no.credentials": "(not connected)",
     "AWS.credentials.statusbar.text": "AWS: {0}",
     "AWS.credentials.statusbar.tooltip": "The current credentials used by the AWS Toolkit.\n\nClick this status bar item to use different credentials.",
-    "AWS.error.during.sam.local": "Failed to run SAM Application locally: {0}",
+    "AWS.error.during.sam.local": "Failed to run SAM application locally: {0}",
```

Comment:
Please make sure all of these strings are also updated in their respective locations in source. Looks like the majority of the changes were not propagated.

(on second look, they may have just been hidden by default in the PR, will take another look through)

Reply:
NVM looks good